homeassistant/components/camera/uvc.py

```diff
@@ -25,12 +25,14 @@
 
 DEFAULT_PASSWORD = 'ubnt'
 DEFAULT_PORT = 7080
+DEFAULT_SSL = False
```

Comment:
Default because the certification is not validate and can't check if the connection is not hacked?

Reply:
I left it at False for several reasons:
1. I made it optional because I didn't want to break people's configuration by having a missing parameter.
1. People with the component configured have it pointing to an http endpoint. If we default to SSL, all those connections will break.
3. Right now the implementation would not support self-signed certificates.